src/js/script-loader.js

```diff
@@ -808,43 +808,52 @@ var LoaderProtoMethods = {
 				} else if (dependency === 'module') {
 					dependencyImplementations.push(moduleImpl);
 				} else if (dependency === 'require') {
-					var localRequire = function(moduleName) {
-						var argc = arguments.length;
-
-						if (argc > 1) {
-							global.require.apply(global.Loader, arguments);
-						} else {
-							moduleName = pathResolver.resolvePath(
-								module.name,
-								moduleName
-							);
-
-							moduleName = configParser.mapModule(
-								moduleName,
-								module.map
-							);
+					// This extra function is necessary to capture the current
+					// loop iteration's module.
+					var createLocalRequire = function(module) {
+						return function(moduleName) {
+							var argc = arguments.length;
+
+							if (argc > 1) {
+								global.require.apply(global.Loader, arguments);
+							} else {
+								moduleName = pathResolver.resolvePath(
+									module.name,
+									moduleName
+								);
 
-							var dependencyModule = configParser.getModules()[
-								moduleName
-							];
-
-							if (
-								!dependencyModule ||
-								typeof dependencyModule.implementation ===
-									'undefined'
-							) {
-								throw new Error(
-									'Module "' +
-										moduleName +
-										'" has not been loaded yet for context: ' +
-										module.name
+								moduleName = configParser.mapModule(
+									moduleName,
+									module.map
 								);
-							}
 
-							return dependencyModule.implementation;
-						}
+								var dependencyModule = configParser.getModules()[
+									moduleName
+								];
+
+								console.log('***********+', moduleName);
```

Comment:
Argghhhh! 😱

The good thing is that we have the stripDebug() task in gulpfile and it doesn't get to production code.

Thx :-).